app.yaml

```diff
@@ -59,6 +59,13 @@ handlers:
   static_files: core/templates/dev/head/\1
   upload: core/templates/dev/head/(.*\.(js))$
   secure: always
+- url: /templates/dev/head/components/(.*\.(html))$
```

Comment:
This is for local dev server `scripts/start.sh`, it's similar to the handler above it. We are not setting cache settings here, we are just serving html files.